src/canvas.js

```diff
@@ -223,8 +223,8 @@ class Canvas {
             }
             try {
               await this.refreshUserToken(this.refreshToken);
-            } catch (err) {
-              console.error(err);
+            } catch (err1) {
```

Comment:
why this change ?

Reply:
Its one of the errors in eslint, its overriding the err, that is declared in the above scope, and was causing this lint issue, so just changed it.